bin/pycbc_condition_strain

```diff
@@ -56,9 +56,15 @@ parser.add_argument('--low-frequency-cutoff', type=float,
                     help='Provide a low-frequency-cutoff for fake strain. '
                          'This is only needed if fake-strain or '
                          'fake-strain-from-file is used')
+parser.add_argument('--frame-duration', type=int,
+                    help='Split all data into smaller frame files of the given duration if specified.')
+
 pycbc.strain.insert_strain_option_group(parser)
 args = parser.parse_args()
 
+if args.frame_duration:
```

Comment:
Hmm, if it is zero, no error will actually be printed. I would use:
```python
if args.frame_duration is not None and args.frame_duration <= 0:
    parser.error('Frame duration should be positive integer, {} given'.format(args.frame_duration))
```
(there is an elegant way of doing this check "properly", but let's leave that for another time and close this issue).